src/main/java/slimeknights/tconstruct/plugin/crt/managers/MaterialManager.java

```diff
@@ -0,0 +1,40 @@
+package slimeknights.tconstruct.plugin.crt.managers;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.api.item.IIngredient;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.api.managers.IRecipeManager;
+import com.blamejared.crafttweaker.impl.actions.recipes.ActionAddRecipe;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraft.util.ResourceLocation;
+import org.openzen.zencode.java.ZenCodeType;
+import slimeknights.tconstruct.library.materials.MaterialId;
+import slimeknights.tconstruct.library.recipe.RecipeTypes;
+import slimeknights.tconstruct.library.recipe.material.MaterialRecipe;
+import slimeknights.tconstruct.plugin.crt.CRTHelper;
+
+@ZenRegister
+@ZenCodeType.Name("mods.tconstruct.Material")
+public class MaterialManager implements IRecipeManager {
```

Comment:
I would name this `MaterialRecipeManager`/`mods.tconstruct.MaterialRecipe` to make it more clear. `Material` implies the tool material JSONs as CT is not just recipes, this recipe adds the mapping from items to materials.